<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>List</class>
 <widget class="QWidget" name="List">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#widget{
	background-image: url(:/pic/background-list.png);
}</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>150</height>
    </rect>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>50</y>
      <width>150</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>39</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
     </font>
    </property>
    <property name="text">
     <string>ID:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>0</y>
      <width>50</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="images.qrc">
      <normaloff>:/pic/iconmonstr-x-mark-5-240.png</normaloff>:/pic/iconmonstr-x-mark-5-240.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>26</height>
     </size>
    </property>
    <property name="autoRaise">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>90</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>幼圆</family>
     </font>
    </property>
    <property name="text">
     <string>个性签名：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>90</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_2">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>90</y>
      <width>30</width>
      <height>20</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>编辑</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="images.qrc">
      <normaloff>:/pic/Edit.png</normaloff>:/pic/Edit.png</iconset>
    </property>
    <property name="autoRaise">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>150</y>
     <width>400</width>
     <height>620</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>400</width>
     <height>620</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>400</width>
     <height>620</height>
    </size>
   </property>
   <property name="toolTip">
    <string/>
   </property>
   <property name="styleSheet">
    <string notr="true">QTabBar::tab{width:200}
QTabBar::tab{height:40}</string>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <property name="iconSize">
    <size>
     <width>25</width>
     <height>25</height>
    </size>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="icon">
     <iconset resource="images.qrc">
      <normaloff>:/pic/icon2/iconmonstr-accessibility-2-240.png</normaloff>:/pic/icon2/iconmonstr-accessibility-2-240.png</iconset>
    </attribute>
    <attribute name="title">
     <string/>
    </attribute>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>400</width>
       <height>100</height>
      </rect>
     </property>
     <property name="text">
      <string>EMPTY</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>120</y>
       <width>400</width>
       <height>100</height>
      </rect>
     </property>
     <property name="text">
      <string>EMPTY</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_5">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>240</y>
       <width>400</width>
       <height>100</height>
      </rect>
     </property>
     <property name="text">
      <string>EMPTY</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_6">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>360</y>
       <width>400</width>
       <height>100</height>
      </rect>
     </property>
     <property name="text">
      <string>EMPTY</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_7">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>470</y>
       <width>400</width>
       <height>100</height>
      </rect>
     </property>
     <property name="text">
      <string>EMPTY</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="icon">
     <iconset resource="images.qrc">
      <normaloff>:/pic/icon2/cp.png</normaloff>:/pic/icon2/cp.png</iconset>
    </attribute>
    <attribute name="title">
     <string/>
    </attribute>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>400</width>
       <height>100</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Comic Sans MS</family>
      </font>
     </property>
     <property name="text">
      <string>Group</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="addfriends">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>770</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>添加好友</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/pic/iconmonstr-friend-2-240.png</normaloff>:/pic/iconmonstr-friend-2-240.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="news">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>770</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>修改密码</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/pic/iconmonstr-lock-5-240.png</normaloff>:/pic/iconmonstr-lock-5-240.png</iconset>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>770</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>设置个人信息</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/pic/icon2/shezhi-b.png</normaloff>:/pic/icon2/shezhi-b.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="theme">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>770</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>换肤</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/pic/icon2/iconmonstr-product-12-240.png</normaloff>:/pic/icon2/iconmonstr-product-12-240.png</iconset>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
